content/blog/precepts.md

```diff
@@ -0,0 +1,355 @@
++++
+title = "Precepts"
+date = 2020-03-14
+description = "Urbit is a principled approach to system design and programming, but it's often not obvious what those technical principles are.  This is a list of the technical maxims that define Urbit's approach to engineering."
```

Comment:
Simplify for the Twitter cards, lol: `The technical maxims that define Urbit's approach to engineering.`

Reply:
good call